src/components/FeeInformation/FeeInformation.tsx

```diff
@@ -0,0 +1,30 @@
+import "./fee-information.sass";
+
+interface Props {
+  feeName: string;
+  amountAsString: string;
+  customPriceColor?: string;
+}
+
+const FeeInformation = ({
+  feeName,
+  amountAsString,
+  customPriceColor,
+}: Props): JSX.Element => {
+  return (
+    <div className="fee-information row">
+      <p className="fee-name">{feeName}</p>
+      <p
+        className={
```

Comment:
actually you may want to use `??` instead of `||` in this example, depending on what behavior you want when `customPriceColor` is an empty string

Reply:
Dude, this is awesome!  I actually thought to do this before seeing this!!!  I didn't do the second part you wrote about using ?? instead of || but I did think to eliminate the duplication of 'product-price' hahaha

Thank you for my first introduction to the nullish coalescing operator!  So inside the interface for Props on this component, I changed the type of customPriceColor to be null | string.  Then I initialized the default value to be null.  So if it's not there, 'black-text' is the string assigned, otherwise whatever the string entered for customPriceColor gets assigned!  In addition to 'product-price' of course!  I realized that the ?? operator did not work when I kept customPriceColor as an empty string because it only works against the null type.